pom.xml

```diff
@@ -21,21 +21,31 @@
     <maven.compiler.parameters>true</maven.compiler.parameters>
     <maven.skip.unit.tests>false</maven.skip.unit.tests>
     <maven.skip.it.tests>false</maven.skip.it.tests>
-    <version.com.fasterxml.jackson.core.jackson-databind>2.10.5.1</version.com.fasterxml.jackson.core.jackson-databind>
+    <version.com.fasterxml.jackson.core.jackson-databind>2.13.2.2</version.com.fasterxml.jackson.core.jackson-databind>
     <version.frontend.plugin>1.12.0</version.frontend.plugin>
-    <version.io.quarkus>2.2.3.Final</version.io.quarkus>
+    <version.io.quarkus>2.7.5.Final</version.io.quarkus>
     <version.io.quarkiverse.file-vault>0.3.0</version.io.quarkiverse.file-vault>
     <version.org.github.tomakehurst.wiremock-jre8>2.27.2</version.org.github.tomakehurst.wiremock-jre8>
     <version.org.kie.server>${project.version}</version.org.kie.server>
     <version.org.mockito>3.7.7</version.org.mockito>
     <version.org.projectlombok.lombok>1.18.16</version.org.projectlombok.lombok>
+    <version.com.h2database>1.4.197</version.com.h2database>
+    <version.org.apache.maven>3.3.9</version.org.apache.maven> <!-- Need to force the version of the maven-core artifact to be in sync with kie. Otherwise it might pick a wrong version coming from quarkus bom import -->
+    <version.org.codehaus.mojo.sql-maven-plugin>1.5</version.org.codehaus.mojo.sql-maven-plugin>
 
     <!-- Quarkus database setup -->
     <maven.jdbc.db-kind>h2</maven.jdbc.db-kind>
-    <maven.jdbc.url>jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE</maven.jdbc.url>
+    <maven.jdbc.url>jdbc:h2:file:~/test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;AUTO_SERVER=TRUE</maven.jdbc.url>
```

Comment:
Is it better to have it in the file?

Reply:
actually, the problem is that we need one single database with the schema already created and persisted somehow across all tests. So if we keep this `mem` it will start up the H2 instance when running the sql-maven-plugin and then [it won't be accessible for any other jvm thread](http://www.h2database.com/html/features.html#in_memory_databases).
So, only option is to use either file based database or use the server mode mechanism in H2 (tcp/ip)

Reply:
Oh, makes sense now 😄